x-pack/plugin/security/qa/operator-privileges-tests/src/javaRestTest/java/org/elasticsearch/xpack/security/operator/Constants.java

```diff
@@ -228,6 +228,7 @@
         "cluster:admin/xpack/watcher/watch/delete",
         "cluster:admin/xpack/watcher/watch/execute",
         "cluster:admin/xpack/watcher/watch/put",
+        "cluster:internal/health",
```

Comment:
The `cluster:internal` namespace today is used only for internal ML actions that run as the privileged `_xpack` user. I don't think this fits there. Are we deliberately avoiding the `cluster:monitor/health*` namespace? If so, why?

Reply:
I went with `cluster:monitor/health/get`. I also debated about `cluster:monitor/health_api`. I'm open to suggestions here. `cluster:monitor/health` is currently taken.

Reply:
I'd lean towards `cluster:monitor/health_api` just to make it clearer that this is a completely separate API and not a "GET" for `_cluster/health`.